governator-core/src/main/java/com/netflix/governator/InjectorBuilder.java

```diff
@@ -62,6 +63,23 @@ public static InjectorBuilder fromModules(List<Module> modules) {
         return new InjectorBuilder(Modules.combine(modules));
     }
     
+    public static <T> InjectorBuilder bindInstance(final Object o) {
```

Comment:
Instead of complicating the InjectorBuilder, this syntactic sugar can better be addressed by moving this into a separate class. 

```java
public static final ModuleUtils {
    public static Module fromInstances(final Object... instances) {
        return new AbstractModule() {
            @SuppressWarnings("unchecked")
            @Override
            protected void configure() {
             	for (Object o : instances) {
             		bind((Class)o.getClass()).toInstance(o);
             	}
         };
    }
}
```

Your unit tests would then look like, 
```java
InjectorBuilder.fromModules(ModuleUtils.fromInstances(foo)).createInjector()
```